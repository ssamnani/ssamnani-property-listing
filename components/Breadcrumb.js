import React from "react";
import "../styles/Breadcrumb.module.css";

function Breadcrumb() {
  return (
    <div className="breadcrumbWrapper">
      <ol>
        <li>Home</li>
        <li>Sale</li>
        <li>All Residential</li>
      </ol>
    </div>
  );
}

export default Breadcrumb;
