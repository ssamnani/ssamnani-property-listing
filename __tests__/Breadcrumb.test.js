import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import Breadcrumb from "../components/Breadcrumb";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render("<Breadcrumb />", div);
});

// Snapshot Testing
it("matches snapshot", () => {
  const tree = renderer.create(<Breadcrumb />).toJSON();
  expect(tree).toMatchSnapshot();
});
