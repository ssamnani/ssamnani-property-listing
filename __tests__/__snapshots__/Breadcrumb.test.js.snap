// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="breadcrumbWrapper"
>
  <ol>
    <li>
      Home
    </li>
    <li>
      Sale
    </li>
    <li>
      All Residential
    </li>
  </ol>
</div>
`;
